core-ui/src/shared/components/EventSubscriptions/EventSubscriptions.js

```diff
@@ -32,7 +33,7 @@ export default function EventSubscriptions({
       namespace,
       ownerRef,
       sink,
-      eventType,
+      EVENT_TYPE_PREFIX + eventType,
```

Comment:
why won't we just add it in the `CreateEventSubscriptionForm` where we have the `EVENT_TYPE_PREFIX` already defined?

Reply:
Because there we use the pure HTML form's field value in the `onSubmit` callback. It is possible but it would complicate things. That's why I decided to put it here.